### PR TITLE
Yuliia/wishlist page main logic

### PR DESCRIPTION
**Pull request by ChebotarYuliia**
1. made a new design for the wishlist page
2. added stepper for wish item creation
3. implemented the logic of editing and deleting a single item

### CODE REVIEW

File: src/pages/WishlistItem.jsx [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+    navigate('/wishlist');
+  };
+
+  useEffect(() => {
+    if (success) {
+      resetStateStatusAndRedirect();
+    }
+  }, [success]);
+
+  const handleTransformButton = async () => {
```
Comment (Bogd1y) [inline src/pages/WishlistItem.jsx:109]: Are you sure that async/await are required here, didn't it work without them?

Comment (ChebotarYuliia) [reply src/pages/WishlistItem.jsx:109]: yup, you're totally right, I don't need them
thank you

File: src/components/forms/CreationForm3.jsx [hunk 2]
```diff
@@ -2,29 +2,28 @@ import { Card, CardMedia, Switch, TextField, Typography } from '@mui/material';
 import { Box, Stack } from '@mui/system';
 import React from 'react';
 import { useSelector } from 'react-redux';
-import { selectBasket} from '../../redux/basket/createBasketSlice'
-import defaultImage from '../../assets/swinka.png'
+import { selectBasket } from '../../redux/basket/createBasketSlice';
+import defaultImage from '../../assets/swinka.png';
 import useModal from '../../hooks/useModal';
```
Comment (ShevtsovVolodymyr) [inline src/components/forms/CreationForm3.jsx:7]:
```suggestion
import { selectBasket } from '@redux/basket/createBasketSlice';
import defaultImage from '@assets/swinka.png';
import useModal from '@hooks/useModal';
```

Comment (ShevtsovVolodymyr) [reply src/components/forms/CreationForm3.jsx:7]: You are able to use alias now if you want to

**Review by ValeriaRbk — APPROVED** [review]

**Review by npelenskii — APPROVED** [review]

**Pull request merged**